examples/kyb-app/src/App.tsx

```diff
@@ -1,21 +1,102 @@
 import '@ballerine/ui/dist/style.css';
-import { RouterProvider } from 'react-router-dom';
-import { router } from '@app/router';
 import { AppLoadingContainer } from '@app/components/organisms/AppLoadingContainer';
 import { useCollectionFlowSchemaQuery } from '@app/hooks/useCollectionFlowSchemaQuery';
+// import { RouterProvider } from 'react-router-dom';
+// import { router } from '@app/router';
+// import { CustomerProvider } from '@app/components/providers/CustomerProvider';
+// import { LoadingScreen } from '@app/common/components/molecules/LoadingScreen';
+// import { CustomerProviderFallback } from '@app/components/molecules/CustomerProviderFallback';
+import { useCustomerQuery } from '@app/hooks/useCustomerQuery';
+import { useUISchemasQuery } from '@app/hooks/useUISchemasQuery';
+import { DynamicUI } from '@app/components/organisms/DynamicUI';
+import { State } from '@app/components/organisms/DynamicUI/StateManager/types';
+import { UIRenderer } from '@app/components/organisms/UIRenderer';
+import { Title } from '@app/components/organisms/UIRenderer/elements/Title';
+import { Cell } from '@app/components/organisms/UIRenderer/elements/Cell';
+import { ActionsHandler } from '@app/components/organisms/DynamicUI/StateManager/components/ActionsHandler';
+import { ButtonUIElement } from '@app/components/organisms/UIRenderer/elements/ButtonUI';
+import { JSONForm } from '@app/components/organisms/UIRenderer/elements/JSONForm/JSONForm';
+import { useFlowContextQuery } from '@app/hooks/useFlowContextQuery';
+import { AnyObject } from '@ballerine/ui';
+import { StepperUI } from '@app/components/organisms/UIRenderer/elements/StepperUI';
+import { useMemo } from 'react';
+import { prepareInitialUIState } from '@app/helpers/prepareInitialUIState';
 import { CustomerProvider } from '@app/components/providers/CustomerProvider';
 import { LoadingScreen } from '@app/common/components/molecules/LoadingScreen';
 import { CustomerProviderFallback } from '@app/components/molecules/CustomerProviderFallback';
-import { useCustomerQuery } from '@app/hooks/useCustomerQuery';
+import { RouterProvider } from 'react-router-dom';
+import { router } from '@app/router';
+
+const elems = {
+  h1: Title,
+  h3: (props: AnyObject) => <h3 className="font-bold">{props?.options?.text}</h3>,
+  'json-form': JSONForm,
+  container: Cell,
+  mainContainer: Cell,
+  'json-form:button': ButtonUIElement,
+  stepper: StepperUI,
+};
 
 export const App = () => {
-  const dependancyQueries = [useCollectionFlowSchemaQuery(), useCustomerQuery()];
+  const dependancyQueries = [useCustomerQuery(), useUISchemasQuery(), useFlowContextQuery()];
```

Comment:
What are we trying to achieve here? Maybe check out react-query's `useQueries`.

Reply:
Good point.I will replace it with useQueries.